src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/[type]/HighRiskBreachLayout.test.tsx

```diff
@@ -89,3 +128,79 @@ it("does not show the Broker step if the user is in a country where the data bro
   const steps = getAllByRole(stepIndicator, "listitem");
   expect(steps).toHaveLength(3);
 });
+
+it("shows the high-risk celebration view, next step is passwords", () => {
+  const ComposedComponent = composeStory(
+    HighRiskBreachDonePasswordsNextStory,
+    Meta,
+  );
+
+  render(<ComposedComponent />);
+
+  const viewHeading = screen.getByRole("heading", {
+    name: "You’ve fixed your high risk exposures!",
+  });
+  expect(viewHeading).toBeInTheDocument();
+
+  const buttonLink = screen.getByRole("link", {
+    name: "Let’s keep going",
+  });
+  expect(buttonLink).toBeInTheDocument();
```

Comment:
We might want to check that the page is actually referring to passwords as the next step? Either via text that's on the page, or that the link is pointing to the correct route?

Reply:
Good idea! Testing the target of the link in [8e32470](https://github.com/mozilla/blurts-server/pull/3753/commits/8e324703efdb631fe290409362e6e12c8b16b748).